python/lsst/obs/pfs/createDetGeom.py

```diff
@@ -0,0 +1,144 @@
+# example: createDetGeom.py
+import os
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+import lsst.utils
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        Task.__init__(self, *args, **kwargs)
+
+    def run(self):
+        
+        """ Gain and RdNoise from Jim Gunn's talk in Marseille Dec 2015"""
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
+        
+        # According to RHL's data model (https://github.com/Subaru-PFS/datamodel/blob/master/datamodel.txt):
+        # The raw CCD images will have:
+        #    NAMP=8
+        #    AMPCOLS=520
+        #    CCDROWS=4224
+        #
+        #    OVERROWS = 76 # as of 2016-02-01
+        #    OVERCOLS = 32 # as of 2016-02-01
+        #
+        #    LEADINROWS = 48 # necked rows
+        #    LEADINCOLS  = 8 # real leadin pixels
+        #
+        # So the raw CCD data will have
+        #    NROW = 4300 #  CCDROWS + OVERROWS
+        #    NCOL = 4416 # NAMP*(AMPCOLS + OVERCOLS)
+        #
+        # And the ISR-extracted CCD images will probably be close to:
+        #    NROW = 4172  # CCDROWS - LEADINROWS
+        #    NCOL = 4096  # NAMP*(AMPCOLS - LEADINCOLS)
+        # This value of NROW is applicable to the pfsArm files.
+
+        #                        according to Craig Loomis:
+        nColsAmp = 512#          512
+        nColsHPrescan = 8#9#     8 + 1 for first, then 8
+        nColsHOverscan = 32#31#  32 - 1 for last, otherwise 32
+        nRowsVPrescan = 48#51#   48-51
+        nRowsVOverscan = 76#75#  76
+        nRowsAmp = 4176#4174#    actual 4224, ignore first ~50
+        saturationLevel = 60000
+        
+        outDir = os.path.join(lsst.utils.getPackageDir('obs_pfs'),'pfs/camera')
+        
+        amp = 0
+        for iArm in range(3):
+            for iCCD in range(4):
+                detector = afwTable.AmpInfoCatalog(afwTable.AmpInfoTable.makeMinimalSchema())
+
+                for iAmp in range(8):
+
+                    x0 = iAmp * (nColsAmp + nColsHPrescan + nColsHOverscan)
+                    x1 = ((iAmp + 1) * (nColsAmp + nColsHPrescan + nColsHOverscan)) - 1
+                    print 'iAmp = ',iAmp,': x0 = ',x0,', x1 = ',x1,', extend = ',x1-x0+1
```

Comment:
1. Please fix the spacing.
2. Do you really need to print so much (here and below)? Would a logger be more appropriate?